providers/windspots.py

```diff
@@ -18,23 +18,23 @@ def process_data(self):
         try:
             self.log.info("Processing WindsSpots data...")
             result = requests.get(
-                "https://api.windspots.com/windmobile/stationinfos?allStation=true",
+                "https://api.windspots.com/windmobile/stationinfo",
                 timeout=(self.connect_timeout, self.read_timeout),
                 verify=False,
             )
 
             for windspots_station in result.json()["stationInfo"]:
                 station_id = None
                 try:
-                    windspots_id = windspots_station["@id"][10:]
+                    windspots_id = windspots_station["id"][10:]
```

Comment:
I think that's windspots is now "winId"!

Reply:
Agree, it seems that you are right :)